toolkit/imageconfigs/core-fips.json

```diff
@@ -5,7 +5,7 @@
             "MaxSize": 2048,
             "Artifacts": [
                 {
-                    "Name": "core",
+                    "Name": "core-fips",
```

Comment:
I apologize, Nick! I know I have asked you to update this name, because I was convinced that's what we were doing with other "core" artifacts, but I later took a look myself and realized, that we've been naming all of them "core" and they were placed into separate image folders anyway, so the artifact name wasn't really the issue. I didn't come back to this, because I though that my request got lost in the e-mail thread anyway, but it looks like it might actually be better to keep it the way it is, since this way we're keeping things consistent and we're causing any real issues.